frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -2149,16 +2282,25 @@ ApplicabilityResult instantiateSignature(Context* context,
                                           fieldType.type(),
                                           fieldType.param()));
 
-      if (shouldIncludeFieldInTypeConstructor(context, fieldDecl->id(), sigType)){
+      if (isFieldSyntacticallyGeneric(context, fieldDecl->id())){
+        // TODO
```

Comment:
Did you mean to write something else here?

Reply:
I don't think so, after all.